src/sentry/integrations/slack/endpoints/action.py

```diff
@@ -194,9 +194,15 @@ def post(self, request: Request) -> Response:
 
         channel_id = slack_request.channel_id
         user_id = slack_request.user_id
+        integration = slack_request.integration
         response_url = data.get("response_url")
 
         if action_option in ["link", "ignore"]:
+            analytics.record(
+                "integrations.slack.chart_unfurl_action",
+                organization_id=integration.organizations.all()[0].id,
```

Comment:
```suggestion
                organization_id=integration.organizations.first().id,
```

Reply:
All the slack endpoints code uses the first pattern, so leaving as is for consistency 🤷‍♀️